src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/compute/v4_9/CloudStackComputePlugin.java

```diff
@@ -165,7 +165,7 @@ void doDeleteInstance(@NotNull DestroyVirtualMachineRequest request,
             LOGGER.info(String.format(Messages.Info.DELETING_INSTANCE, instanceId, token));
         } catch (HttpResponseException e) {
             LOGGER.error(String.format(Messages.Error.UNABLE_TO_DELETE_INSTANCE, instanceId), e);
-            CloudStackHttpToFogbowExceptionMapper.map(e);
+            throw CloudStackHttpToFogbowExceptionMapper.get(e);
```

Comment:
Please keep using the map method to keep pattern. You have to fix the map method instead of using this get one, for more details check OpenStackHttpToFogbowExceptionMapper.

Reply:
We debated it in the last meeting.

Reply:
Check the **CloudStackHttpToFogbowExceptionMapper** class apparently the _map_ method has not been updated to _get_ and is causing conflict.